remotespark/MessagePrinter.py

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2015  aggftw@gmail.com
```

Comment:
Also, I've created a very similar class... Let's work on merging our classes. Let me check it in and show you how to merge it.